mobile/lib/services/rest_api.dart

```diff
@@ -256,7 +256,7 @@ class AirqoApiClient {
 
   Future<void> syncPlatformAccount() async {
     try {
-      await Future.delayed(const Duration(minutes: 5));
+      await Future.delayed(const Duration(minutes: 1));
```

Comment:
Hi @BenjaminSsempala , I have mixed feelings about this "wait" algorithm.
Why do we have to wait? Can't we have things instant?
Could we have a brief discussion about this later today?

Reply:
Thanks @Baalmart I think the wait is a small compromise for efficiency.

1.  At this exact point, the user is finishing up the signup process in the mobile app. This is similar to putting this step after the user has finished the signup and onboarding screens. In other words, the user will still continue using the mobile app in the same order.
2. It allows the emails to be able to use the name of the user.
3. It also allows the name of the user to be added to their platform account, at the point of account creation.

Reply:
thanks @BenjaminSsempala